cmd/monitoring/monitoring.go

```diff
@@ -238,6 +254,22 @@ func loadOCConfig() (*api.OpenShiftManagedCluster, error) {
 	return cs, nil
 }
 
+func (m *monitor) trackCreationTimestamp() {
```

Comment:
I don't think this stuff is right - running monitoring != creating a cluster
we can add this kind of instrumentation in the fakerp separately